addons/semicolon/semicolon.js

```diff
@@ -1,14 +1,14 @@
 export default async function ({ addon, global, console }) {
-  if (document.body.contains(document.querySelector("#signature"))) {
-    console.log("Signature already existed");
-  } else {
-    if (document.location.href.split("/")[5] == "editor") {
-      console.log("In editor");
+    if (document.body.contains(document.querySelector("#signature"))) {
```

Comment:
Wut? Weird use of `contains` there. Why not just use a selector? Like `document.querySelector("body #signature")`